WordPress/gutenberg
# [Navigation link block][Try] Show a URL next to the link icon

## Description

As in the title. Solves #22836.

**Before:**

<img width="687" alt="Zrzut ekranu 2020-06-9 o 13 28 21" src="https://user-images.githubusercontent.com/205419/84142457-1802cc00-aa55-11ea-87c4-d403f53aa454.png">

**After:**

![2020-06-24 15-45-02 2020-06-24 15_45_52](https://user-images.githubusercontent.com/205419/85568280-d61b7d80-b631-11ea-8986-fc8842862970.gif)

## How has this been tested?
1. Enable the navigation experiment in Gutenberg > Experiments
1. Go to Gutenberg > Navigation (beta)
1. Add a menu item with no link, confirm there is no text next to the link icon in the toolbar
1. Attach a link to that menu item, confirm there link shows up next to the link icon in the toolbar

## Types of changes
New feature (non-breaking change which adds functionality)

## Checklist:
